src/Microsoft.ManagedZlib/tests/MyClassTests.cs

```diff
@@ -0,0 +1,339 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Sdk;
+
+namespace Microsoft.ManagedZLib.Tests;
+
+public class MyClassTests
```

Comment:
I think this class and its filename should be renamed to `DeflateStreamTests` to clearly indicate that this is the only type were' testing.
```suggestion
public class DeflateStreamTests
```